recipe/meta.yaml

```diff
@@ -8,16 +8,18 @@ source:
   path : ..
 
 build:
-  number: 1
+  number: 2
   noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - lucid = lucid.launcher:main
+    - QSGenerateExcelSpreadSheet.py = psdm_qs_cli.QSGenerateExcelSpreadSheet:main
+    - QSGenerateJSON.py = psdm_qs_cli.QSGenerateJSON:main
 
 requirements:
   build:
     - python
-    - setuptools
-    - krtc
-    - requests
-    - six
```

Comment:
Maybe those should go as `run` dependencies?

Reply:
They are - the github diff just hides it though

Reply:
Oh, ok! Sorry about that.